Rater.API/appsettings.json

```diff
@@ -2,6 +2,11 @@
   "ConnectionStrings": {
     "DefaultConnection": "Host=db;Port=5432;Database=postgres;Username=admin;Password=pass"
   },
+
+  "jwt": {
+    "Token": "75e3d0b0db6c6a616472953fc4a63921484644e92f34f0867f4500be96576c803ed2405828712c4ed8be43eec1a246915682e0e96b6b0b231113c4772530a761"
```

Comment:
We might want to keep this in a secret for the deployment pipelines

Reply:
Since the project is currently running locally, storing the JWT token here shouldn't be a problem. However, please consider creating an issue for this potential concern.

Reply:
OK, you can merge